profiles/ug/modules/ug/ug_system/ug_system.install

```diff
@@ -1,21 +1,163 @@
 <?php
 
 /**
- * Implements hook_requirements().
- */
+* Implements hook_requirements().
+**/
 function ug_system_requirements($phase) {
   $requirements = array();
   if ($phase == 'runtime') {
+    // ------------------------
+    // UPDATE NOTIFY EMAILS
+    // ------------------------
     $update_notify_emails = variable_get('update_notify_emails', array());
     $requirements['ug_system_update_notify_emails'] = array(
-      'title' => t('Update notify emails'),
+      'title' => t('UG Update notify emails'),
       'value' => implode(', ', $update_notify_emails),
     );
     if (in_array('admin@example.com', $update_notify_emails)) {
       $requirements['ug_system_update_notify_emails']['description'] = t('Update notification email has not been changed from its default value.');
       $requirements['ug_system_update_notify_emails']['severity'] = REQUIREMENT_ERROR;
     }
-  }
-  return $requirements;
-}
 
+    // ------------------------
+    // SITE MAIL
+    // ------------------------
+    $site_mail = variable_get('site_mail');
+    $requirements['ug_system_site_mail'] = array(
+      'title' => t('UG Site e-mail'),
+      'value' => check_plain($site_mail),
+      'description' => t('This checks if site email is changed from no-reply@uoguelph.ca'),
+    );
+    if ($site_mail == 'no-reply@uoguelph.ca') {
+      $requirements['ug_system_site_mail']['description'] = t('The site mail must be changed.');
+      $requirements['ug_system_site_mail']['severity'] = REQUIREMENT_ERROR;
+    }
+
+    // ------------------------
+    // RECAPTCHA SITE KEY
+    // ------------------------
+    $recaptcha_site_key = variable_get('recaptcha_site_key');
+    $requirements['ug_system_recaptcha_site_key'] = array(
+      'title' => t('UG recaptcha Site key'),
+      'description' => t('This ensures the recaptcha site key is correct'),
+    );
+    if ($recaptcha_site_key != '6LfhO_gSAAAAAL4stbeWkKhUsmgs2u68CvBxk53M') {
```

Comment:
We can't commit this key to the code repository. You could just test to see of the key has been set.